tools/accuracy_checker/accuracy_checker/postprocessor/README.md

```diff
@@ -56,7 +56,11 @@ Accuracy Checker supports following set of postprocessors:
 * `clip_segmentation_mask` - clipping segmentation mask values. Supported representations: `BrainTumorSegmentationAnnotation`, `BrainTumorSegmentationPrediction`.
   * `min_value` - lower bound of range.
   * `max_value` - upper bound of range.
-* `segmentation-prediction-resample` - resamples output prediction in two steps: 1) resizes it to bounding box size; 2) extends to annotation size. Supported representations: `BrainTumorSegmentationAnnotation`, `BrainTumorSegmentationPrediction`. For correct bounding box size must be set via tag `boxes_file` in `brats_numpy` [converter](../annotation_converters/README.md).
+* `segmentation_prediction_resample` - resamples output prediction in two steps: 1) resizes it to bounding box size; 2) extends to annotation size. Supported representations: `BrainTumorSegmentationAnnotation`, `BrainTumorSegmentationPrediction`. For correct bounding box size must be set via tag `boxes_file` in `brats_numpy` [converter](../annotation_converters/README.md) or `crop_brats` [preprocessor](../preprocessor/README.md).
```

Comment:
> For correct bounding box size must be set via tag `boxes_file` in `brats_numpy` 

 is it still applicable?

Reply:
Yes, but also works with `crop_brats` preprocessor. Thanks for remark!